app/logic/adminNewEvent.py

```diff
@@ -0,0 +1,63 @@
+from app.models.event import Event
+from datetime import *
+# from datetime import timedelta
+from app.models.facilitator import Facilitator
+# from dateutil import parser
+
+def calculateRecurringEventFrequency(recurringEventInfo):

```

Comment:
This function does not return anything. I'm not sure how you can use it.

Reply:
It is being used in a separate branch/PR. we created it before splitting the issues up. The function has been deleted from this branch to avoid confusion.